src/hooks/filters.ts

```diff
@@ -122,44 +118,42 @@ export function useFilters(queries: OracleQueryUI[]) {
   };
 }
 
+function countQueriesByFilter(
+  queries: OracleQueryUI[],
+  filterName: FilterName,
+  itemName: string
+) {
+  return queries.filter((query) => query[filterName] === itemName).length;
+}
+
 function filtersReducer(queries: OracleQueryUI[]) {
   // use currying to capture queries
   return function reducer(state: State, action: Action) {
     switch (action.type) {
       case "make-entries": {
         const newState = cloneDeep(state);
 
+        newState.filters.project.All.count = queries.length;
```

Comment:
this logic was causing double counting. This new implementation is much simpler and fixes the bug. We don't need to iteratively count items, we can just get the length once and be done with it